test/failure_handler/src/share/conf/linux.properties

```diff
@@ -115,6 +117,15 @@ memory.vmstat.default.args=3 3
 memory.vmstat.statistics.args=-s
 memory.vmstat.slabinfo.args=-m
 memory.vmstat.disk.args=-d
+memory.proc_meminfo.app=bash
+memory.proc_meminfo.args=-c\0cat /proc/meminfo
+memory.proc_meminfo.delimiter=\0
+memory.proc_vmstat.app=bash
+memory.proc_vmstat.args=-c\0cat /proc/vmstat
+memory.proc_vmstat.delimiter=\0
+memory.thp.app=bash
+memory.thp.args=-c\0cat /sys/kernel/mm/transparent_hugepage/{enabled,defrag}
```

Comment:
I think we should include `shmem_enabled` and we could use `tail` if we want to see what files have what info. But I don't have a strong feeling about that:
```
$ cat /sys/kernel/mm/transparent_hugepage/{enabled,defrag,shmem_enabled}
always [madvise] never
always defer defer+madvise [madvise] never
always within_size advise [never] deny force
```
Versus:
```
$ tail /sys/kernel/mm/transparent_hugepage/{enabled,defrag,shmem_enabled}
==> /sys/kernel/mm/transparent_hugepage/enabled <==
always [madvise] never

==> /sys/kernel/mm/transparent_hugepage/defrag <==
always defer defer+madvise [madvise] never

==> /sys/kernel/mm/transparent_hugepage/shmem_enabled <==
always within_size advise [never] deny force
```

Reply:
I added shmem_enabled but kept cat for now.